Makefile

```diff
@@ -38,6 +38,14 @@ get-plugin-dependencies: go.mod go.sum
 compare-deps: get-plugin-dependencies $(GLOOE_DIR)/dependencies
 	go run scripts/compare_deps/main.go plugin_dependencies $(GLOOE_DIR)/dependencies
 
+#----------------------------------------------------------------------------------
```

Comment:
Ideally, the standard workflow is just to run this new, improved `resolve-deps` target. We could support a dry-run flag if it was desirable to skip the automated resolution. So I think as part of our work on this PR we can deprecate `compare-deps`.